gunicorn/management/commands/run_gunicorn.py

```diff
@@ -19,7 +19,7 @@
 try:
     from django.db.backends import BaseDatabaseWrapper, DatabaseError
 
-    if "validate_thread_sharing" in BaseDatabaseWrapper.__dict__:
+    if "validate_thread_sharing" in vars(BaseDatabaseWrapper):
```

Comment:
`run_gunicorn` has been deprecated and it will be removed in Gunicorn 20. It would be better to keep it as is.